### DISCUSSION TITLE
Update Zigbee2MQTT from 1.28.4 to 1.32.2

### DISCUSSION THREAD

**Discussion by Anjerlaan** [Koenkk/zigbee2mqtt#18669] [General]
Hi,

It has been a while since I have updated Zigbee2MQTT on my Raspberry Pi 3b+ with Raspian Buster.
Node.js, is already upgraded to 16
Are there any preparations I need to perform before I update from version 1.28.4 to 1.32.2?

Greetzzz,

Gerben

**Comment by Koenkk** [comment]
> Are there any preparations I need to perform before I update from version 1.28.4 to 1.32.2?

No, just update 😄
